themes/src/main/resources/theme/keycloak.v2/login/template.ftl

```diff
@@ -27,7 +27,8 @@
     <script type="importmap">
         {
             "imports": {
-                "alpinejs": "${url.resourcesCommonPath}/node_modules/alpinejs/dist/module.esm.js"
+                "alpinejs": "${url.resourcesCommonPath}/node_modules/alpinejs/dist/module.esm.js",
```

Comment:
we no longer use alpinejs
```suggestion
```

Reply:
This is for v25, which still uses Alpine.js :)